openapi/mx_platform_api_beta.yml

```diff
@@ -2673,7 +2673,7 @@ paths:
     post:
       description: Use this endpoint to fetch the statements associated with a particular
         member.
-      operationId: fetchStatementsByMember
+      operationId: fetchStatements
```

Comment:
Same here.

Reply:
In certain scenarios like our premium agg jobs, we decided we will have outliers that don't follow our rule above.